system/assets/js/admin/specific/datamanager/quickAddForm/quickAddFormHandler.js

```diff
@@ -72,6 +72,9 @@
 
 	resetForm = function(){
 		$quickAddForm.trigger( "reset" );
+		for (instance in CKEDITOR.instances){
```

Comment:
what if `CKEDITOR.instances` does not exist (i.e. there are no richeditors in the form), will this produce a js error (I suspect so)? Also, is there a possibility that there are ckeditor instances outside of the quick add form? I'm guessing *not* as I believe the form is in an iframe but worth checking for too. Please add some defensive logic for that.

Finally, it would make sense to refactor this out to its own mthod:

```
clearRicheditors( $quickAddForm );
```